pkg/test/constants/constants.go

```diff
@@ -45,7 +45,7 @@ var TestNameRegexesToSkip = []string{
 	".*(containerattachedcluster).*",
 	// Disable due to TF bug https://github.com/hashicorp/terraform-provider-google/issues/16255.
 	// We can't specify labels in the create operation, that causes AssertLabelsMatchAndHaveManagedLabel check to fail.
-	".*(privateserviceconnectforwardingrule).*",
```

Comment:
Could you help me understand why do we want to make this change?

Reply:
privateserviceconnectforwardingrule is renamed to regionalforwardingrulepsc

Reply:
Gotcha! I got confused because I found the sample is renamed to `regional-forwardingrule-vpc-psc` but didn't realize this is the name for testdata.